docs/BindingsOverview.md

```diff
@@ -168,3 +169,80 @@ e.g.
 ```sql
 UPDATE [Products].[az_func].[Leases_<FunctionId>_<TableId>] SET _az_func_AttemptCount = 0 WHERE ProductId = 123
 ```
+
+#### Lease Tables clean up
+
+Before clean up, please see [Leases table](./TriggerBinding.md#az_funcleases_) documentation for how to get the correct Leases table.
+
+- Clean up all the tables not accessed more than N number of days, Please initialize N according to your use case:
+
+```sql
+DECLARE @table_name NVARCHAR(MAX);
+DECLARE @UserFunctionId char(16);
+DECLARE @UserTableId int;
+DECLARE @N int;
+DECLARE leasetable_cursor CURSOR FOR
+
+SELECT 'az_func.Leases_'+UserFunctionId+'_'+convert(varchar(100),UserTableID) as TABLE_NAME, UserFunctionID, UserTableID
+FROM az_func.GlobalState
+WHERE DATEDIFF(day, LastAccessTime, GETDATE()) > N
+
+OPEN leasetable_cursor;
+
+FETCH NEXT FROM leasetable_cursor INTO @table_name, @UserFunctionId, @UserTableId;
+
+WHILE @@FETCH_STATUS = 0
+BEGIN
+    EXEC ('DROP TABLE IF EXISTS ' + @table_name);
+    DELETE FROM az_func.GlobalState WHERE UserFunctionID = @UserFunctionId and UserTableID = @UserTableId
+    FETCH NEXT FROM leasetable_cursor INTO @table_name, @UserFunctionId, @UserTableId;
+END;
+
+CLOSE leasetable_cursor;
+
+DEALLOCATE leasetable_cursor;
+```
+
+- Clean up a specific lease table concerning a specific function:
+
+To find the name of the leases table associated with your function, look in the log output for a line such as this which is emitted when the trigger is started.
+
+`SQL trigger Leases table: [az_func].[Leases_84d975fca0f7441a_901578250]`
+
+This log message is at the `Information` level, so make sure your log level is set correctly.
+
+```sql
+DECLARE @table_name NVARCHAR(MAX); --@table_name would be [az_func].[Leases_84d975fca0f7441a_901578250]
+DECLARE @UserFunctionId char(16); -- @UserFunctionId would be 84d975fca0f7441a
+DECLARE @UserTableId int; -- @UserTableId would be 901578250
+
+DROP TABLE IF EXISTS @table_name
+DELETE FROM az_func.GlobalState WHERE UserFunctionID = @UserFunctionId and UserTableID = @UserTableId
+```
+
+- Clear all trigger related data for a reset:
```

Comment:
Maybe just have the first script be able to take in -1 for the days value to clean up all tables instead? Help keep things simpler